test/test_p2p.py

```diff
@@ -8,35 +8,70 @@ class TestPeerToPeer(unittest.TestCase):
     def setUp(self):
         self.client, self.server = bus.Bus.socketpair(attach_event=True)
 
-        @bus.Object.interface('cockpit.Test')
-        class TestObject(bus.Object):
-            @bus.Object.property('i')
-            def answer(self):
-                return 42
+        class cockpit_Test(bus.Object):
+            answer = bus.Interface.Property('i', value=42)
```

Comment:
`bus.Object.Property` works as well here, hmm...

Reply:
Ya.  Object inherits Interface.  It's weird, I know...